change/@microsoft-teams-js-856c0e77-b263-4d92-ba73-eeb35a3c66b1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Modified prepBetaRelease script",
```

Comment:
>Modified prepBetaRelease script [](http://example.com/codeflow?start=14&length=31)

I know that this doesn't matter as much for external people reading the changelog but you might consider elaborating just a bit, e.g. "Modified prepBetaRelease script to use proper number comparison instead of string comparison for numbers."

Reply:
This won't actually be added to the changelog due to the "none" status, hence why I elected to keep this short and instead elaborate more on the PR description itself.